gdcdictionary/schemas/diagnosis.yaml

```diff
@@ -731,6 +750,8 @@ properties:
       - Stage IIC
       - Stage III
       - Stage IIIA
+      - Stage IIIA1
```

Comment:
DICT93 only lists Stage IIIA2 as the new enum. This doesn't _seem_ wrong though - just verifying.

Reply:
@tlicht3 added that along with StageIIIA2